server/app/api/controllers/self_review.js

```diff
@@ -123,11 +122,12 @@ module.exports = {
     Self_Review_Model.findOneAndUpdate(
       { _id: req.params.id },
       {
-        status: "Inactive"
+        status: "Inactive",
+        updated_date : new Date(),
+        last_updated_by : req.user.userName
       },
-      function (err, userInfo) {
+      function (err, info) {
```

Comment:
info

Reply:
Info was assumed as part of syntax as call back function. Removed as it's not being used and api is working fine without info parameter.

Reply:
Updated same change for all working api's